aws/src/integration/java/org/apache/iceberg/aws/glue/TestGlueCatalogTable.java

```diff
@@ -258,10 +247,10 @@ public void testRenameTableFailsToCreateNewTable() {
         .hasMessageContaining("Table already exists");
     // old table can still be read with same metadata
     Table oldTable = glueCatalog.loadTable(id);
-    Assert.assertEquals(table.location(), oldTable.location());
-    Assert.assertEquals(table.schema().toString(), oldTable.schema().toString());
-    Assert.assertEquals(table.spec(), oldTable.spec());
-    Assert.assertEquals(table.currentSnapshot(), oldTable.currentSnapshot());
+    assertThat(oldTable.location()).isEqualTo(table.location());
+    assertThat(oldTable.schema()).asString().isEqualTo(table.schema().toString());
```

Comment:
same

Reply:
As commented above, it's necessary because the testing the comparisons between objects fail.